components/html-block/html-block.js

```diff
@@ -201,6 +213,30 @@ class HtmlBlock extends LitElement {
 		return false;
 	}
 
+	async _render(elem) {
```

Comment:
I might be inclined to name this differently, since this is specifically for calling the delegated renderers, or whatever we want to call them.  Not sure I have a better name in mind at the moment.  The reason I was thinking of this is because the if-else condition above (for calling this._stamp vs this._renderInline) is repeated in two places, and I was _almost_ going to suggest factoring it out into a render method.

Reply:
Yeah, I was musing about that myself. I was a bit torn on the idea of combining them into one function to avoid repetition and make potential later changes a bit more pleasant. I'll think a bit about how to name this a bit better and incorporate that.